Dockerfile

```diff
@@ -14,6 +14,7 @@ WORKDIR /
 
 FROM base as production
 ENTRYPOINT ["poetry", "run", "gunicorn", "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
+##CMD poetry run gunicorn "todo_app.app:create_app()" bind 0.0.0.0:$PORT
```

Comment:
Hmm, this looks like you're running different commands on Azure vs Heroku - we should find one that works for both (I'd expect this commented one should?)